prepare_ui_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+### Download Docker Images
+echo "you must have permission to run docker."
+docker pull selenoid/vnc_chrome:80.0
+docker pull selenoid/video-recorder:latest-release
+docker pull aerokube/selenoid-ui
```

Comment:
It can be `docker-compose pull` – it can pull several images in parallel.

Reply:
vnc_chrome and video-recorder is not really part of compose, it is being used by selenoid container, so docker-compose pull will only pull selenoid-ui and selenoid, so this will probably be needed.